src/test/resources/omop/concept-ancestor-table-data.json

```diff
@@ -1,2 +1,7 @@
 {"descendant_concept_id": 1, "ancestor_concept_id": 2}
```

Comment:
We could use the same dataset for the set up script if we want, but this is a very small dataset with only three concepts. It is good for the connected and integration tests, but maybe wouldn't be the best for for the set-up script just because it isn't very robust. But maybe it would work fine. Do you mean removing the data used for the set up script and re-using this data?

Reply:
It seems a little out of scope of this ticket to update the populate_omop_data script. I don't think having two sets of omop data is necessarily a problem, and they don't need to be kept in sync with or updated with one another. But I agree, it would be nice to have one set of data for the tests and the set up script. So, should I make a tech debt ticket to address this?

Reply:
> So, should I make a tech debt ticket to address this?